XRTK-Core/Packages/com.xrtk.core/Interfaces/Providers/Controllers/IMixedRealityHandController.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using XRTK.Definitions.Utilities;
+using XRTK.Providers.Controllers.Hands;
+
+namespace XRTK.Interfaces.Providers.Controllers
+{
+    /// <summary>
+    /// Controller definition, used to manage a hand controller.
+    /// </summary>
+    public interface IMixedRealityHandController : IMixedRealityController
+    {
+        /// <summary>
+        /// Get the hands bounds of a given type, if they are available.
+        /// </summary>
+        /// <param name="handBounds">The requested hand bounds.</param>
+        /// <param name="bounds">The bounds if avaialble.</param>
+        /// <returns>True, if bounds available.</returns>
+        bool TryGetBounds(TrackedHandBounds handBounds, out Bounds[] bounds);
+
+        /// <summary>
+        /// Get the current pose of a joint of the hand.
+        /// </summary>
+        /// <remarks>
+        /// Hand bones should be oriented along the Z-axis, with the Y-axis indicating the "up" direction,
+        /// i.e. joints rotate primarily around the X-axis.
+        /// </remarks>
+        bool TryGetJointPose(TrackedHandJoint joint, out MixedRealityPose pose);
+
+        /// <summary>
+        /// Updates the hand controller using mapped hand data.
+        /// </summary>
+        /// <param name="handData">Platform agnostics hand controller data.</param>
+        void UpdateController(HandData handData);
```

Comment:
I'm not sure this needs to be part of the interface, does it?

Reply:
My concern is that something outside of the hand's controller data provider will try to call this

Reply:
Good point. Removed from interface.

Reply:
Removed in my #496 CR